notebooks/wps_decompose_flow_vectors_demo.ipynb

```diff
@@ -34,7 +34,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Using thunderbird on https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps\n"
+      "Using thunderbird on https://marble-dev01.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps\n"
```

Comment:
The host is subject to change (you just did it). Is there some way for this to be set programmatically, or from an env var (THREDDS_HOST)?

Reply:
It's also possible this file is the output of some automated process, if so, ignore my comment.